decidim-core/config/locales/en.yml

```diff
@@ -326,6 +326,7 @@ en:
     filters:
       linked_classes:
         all: All
+        collaborativedraft: Collaborative draft
```

Comment:
Yes, but as the line bellow `dummyresource` also it should be `dummy_resource`, that is because of the `decidim-core/app/helpers/decidim/resource_helper.rb` does a `downcase` instead of an `underscore`:

https://github.com/decidim/decidim/blob/a58eb460b868166d515ef313c6f8bc3168ab353e/decidim-core/app/helpers/decidim/resource_helper.rb#L47-L48

Should I fix  `linked_classes_for`?

Reply:
Ok, I'll fix it

Reply:
I've done a PR to fix it: https://github.com/decidim/decidim/pull/3944

Reply:
Merged with `master` and fixed here